zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/identity/AuthorizationCheckBehavior.java

```diff
@@ -519,6 +519,10 @@ private List<PersistedMapping> getPersistedMappings(final TypedRecord<?> command
         .toList();
   }
 
+  public boolean mappingExists(final String mappingId) {
```

Comment:
❓ in my idea this method is not very relevant to AuthorizationCheckBehaviour, what do you think about injecting the MappingState in PermissionBehaviour and use it directly there in mappingExists method?